app/views/internal/response_templates/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= paginate @response_templates %>
+<div class="wrapper" style="border-bottom: 1px solid black;">
```

Comment:
In general, let's try to rely on bootstrap whenever possible and avoid using inline styles in internal. This will make it easy when we inevitably do a redesign for Phase 2.